custom-vu/src/main/kotlin/jces1209/vu/ConfigProperties.kt

```diff
@@ -0,0 +1,74 @@
+package jces1209.vu
+
+import java.io.InputStream
+import java.util.*
+
+class ConfigProperties(
+    val createIssue: Int,
+    val workAnIssue: Int,
+    val manageProjects: Int,
+    val projectSummary: Int,
+    val browseProjects: Int,
+    val browseBoards: Int,
+    val viewBoard: Int,
+    val workOnDashboard: Int,
+    val workOnSprint: Int,
+    val browseProjectIssues: Int,
+    val workOnBacklog: Int,
+    val workOnSearch: Int,
+    val workOnTopBar: Int,
+    val bulkEdit: Int,
+    val workOnTransition: Int,
+    val workOnWorkflow: Int,
+    val browseWorkflows: Int,
```

Comment:
I am not familiar with kotlin, but I'm just wondering if this kind of long parameter list is a code smell that should be improved?